packages/gasket-typescript-tests/test/log.spec.ts

```diff
@@ -24,9 +24,8 @@ describe('gasket/log', function () {
       logger.emerg('message');
       logger.error('message');
       logger.info('message');
-      logger.warning('message');
+      logger.warn('message');
```

Comment:
So, are we keeping `@gasket/log` around? Or is it no longer necessary and going away with the `log` plugin?

If we are keeping it (and I don't know that we should) then it would need to confirm to the minimum log levels changes.

Reply:
Seems like we won't need `gasket/log` anymore?